modules/KIWILinuxRC.sh

```diff
@@ -3162,6 +3162,14 @@ EOF
     else
         echo "GRUB_TERMINAL=console"  >> $inst_default_grub
     fi
+
+    # Set serial console mode for ec2* firmwares. This is a hack for
+    # Azure images in order to avoid new attributes in XML schema.
+    if [[ "$kiwi_firmware" =~ "ec2" ]]; then
```

Comment:
Also needs the format check for vhdfixed.

Reply:
Here we do not have any $kiwi_format variable available, checking how it can be included here is needed for this change.